app/src/main/res/values/strings.xml

```diff
@@ -4,7 +4,7 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <resources>
     <!-- Name of the application -->
-    <string name="app_name">Fenix</string>
+    <string name="app_name">%1$s</string>
```

Comment:
I don't believe this works. This string is used as the title of the application in all places where it's used. I don't believe you can parameterize it, but it is possible to use a different string based upon the application flavor.